website/pages/docs/k8s/connect/ingress-gateways.mdx

```diff
@@ -5,4 +5,175 @@ sidebar_title: Ingress Gateways
-# Ingress Gateways on Kubernetes
\ No newline at end of file
+# Ingress Gateways on Kubernetes
+
+-> 1.8.0+: This feature is available in Consul versions 1.8.0 and higher
+
+~> This topic requires familiarity with [Ingress Gateways](https://www.consul.io/docs/connect/ingress-gateway).
+
+This page describes how to enable external access to Connect service mesh services running inside Kubernetes using Consul ingress gateways.
+See [Ingress Gateways](https://www.consul.io/docs/connect/ingress-gateway) for more information on use-cases and how it works.
+
+Adding an ingress gateway is a multi-step process, that at a high level, consists of the following steps:
+
+* Setting the helm chart configuration
+* Deploying the helm chart
+* Configuring the gateway
+* Defining an Intention (optional)
+* Deploying your application to Kubernetes
+* Connecting to your application
+
+## Setting the helm chart configuration
+Helm options
+When deploying the helm chart you must provide helm with a custom values.yaml file that contains your environment configuration.
+The following yaml snippet is the launching point for a valid configuration that must be supplied when installing using the [official consul-helm chart](https://hub.helm.sh/charts/hashicorp/consul).
+Information on additional options can be found [here](https://www.consul.io/docs/k8s/helm). Configuration options for ingress gateways reside under the [ingressGateways](https://www.consul.io/docs/k8s/helm#v-ingressgateways) entry.
+
+```shell-session
+$ cat > config.yaml <<EOF
+connectInject:
+  enabled: true
+ingressGateways:
+  enabled: true
+  gateways:
+    - name: ingress-gateway
+      replicas: 2
+      service:
+       type: LoadBalancer
+EOF
+```
+
+The gateways stanza is where you will define and configure the set of ingress gateways you want deployed to your environment.
+The only required field for each entry is `name`, though entries may contain any of the fields found in the `defaults` stanza.
+Values in this section override the values from the defaults stanza for the given ingress gateway with one exception.
+The annotations from the defaults stanza will be appended to any user-defined annotations defined in the gateways stanza.
+Please refer to the ingress gateway configuration [documentation](https://www.consul.io/docs/k8s/helm#v-ingressgateways-defaults) for a detailed explanation of each option.
+
+-> Note: Make sure any ports that will be used as listeners in the ingress gateway's Consul config entry are included in the `ports` object for each gateway.
+
+## Deploying the helm chart
+
+Ensure you have the latest consul-helm chart by running the following command:
+
+```shell-session
+ $ helm repo add hashicorp https://helm.releases.hashicorp.com
+```
+
+Assuming you saved your configuration to a file called config.yaml in the local directory, you can apply your configuration with the following command:
+
+```shell-session
+$ helm install -f config.yaml consul hashicorp/consul --wait
```

Comment:
```suggestion
$ helm install -f config.yaml <consul-release> hashicorp/consul --wait
```
This makes it match the later commands. Otherwise, the uninitiated might not realize they can name this whatever they want or that it is the value to supply in the later commands.